### PR TITLE
Adds hyak and qsiprep pages

### PR DESCRIPTION
**Pull request by teresamg**
First draft

@mckenziephagen

### CODE REVIEW

File: infra/hyak.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+More information on modules can be found [here](https://hyak.uw.edu/docs/tools/modules/)
+
+## Submitting jobs to slurm
+
+You can also submit a task to be queued and executed through slurm. 
+
+The status of these jobs, as well as interactive sessions, can be seen with:
+`squeue -u [UW Net ID]`
+
+## Memory management
```
Comment (arokem) [inline infra/hyak.md]:
```suggestion
## Managing your disk space 
```

File: infra/hyak.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The home directory you begin in after logging in has limited memory. More (but not persistent) memory can be accessed by moving large files, caches, and working directories to /gscratch. For example:
```
Comment (arokem) [inline infra/hyak.md]:
```suggestion
The home directory you begin in after logging in has limited space. More (but not persistent) space can be accessed by moving large files, caches, and working directories to /gscratch. For example:
```

**Review by arokem — COMMENTED** [review]
Looks great! My one comment is to use "space" to describe disk-space allocation, instead of "memory"

**Comment by mckenziephagen** [comment]
looks good to me!

**Comment by arokem** [comment]
I think that I've learned some more things about how to work on hyak and manage the rather limited quota we have in our home directories, so I'll go ahead and merge and then make a separate PR with some proposed additions/changes.

**Pull request merged**